cpp/src/Ice/SHA1.cpp

```diff
@@ -166,3 +166,19 @@ IceInternal::sha1(const unsigned char* data, size_t length, vector<unsigned char
     ::SHA1(&data[0], length, &md[0]);
 #endif
 }
+
+void
+IceInternal::sha1(const byte* data, size_t length, vector<byte>& md)
```

Comment:
Would we remove the `unsinged char` overload later?

Reply:
Yes!